src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -339,6 +339,13 @@ public function onPullAfterCreate(Event $event)
 			// Check the Categories based on the files that gets changed
 			$this->checkCategories($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
 		}
+
+		// Detect first time contributiors...
+		if ($hookData->pull_request->author_association === 'FIRST_TIME_CONTRIBUTOR' || $hookData->pull_request->author_association === 'FIRST_TIMER')
```

Comment:
I cannot find `author_association` property in [Pulls v3 API](https://developer.github.com/v3/pulls/). Am I missing something?